It is simply a code quality issue:
```pytb
AssertionError: File contains trailing whitespace: /home/travis/virtualenv/python2.6.9/lib/python2.6/site-packages/sympy/plotting/tests/test_plot.py, line 94.
```

Do you plan to fix that?

--- a/sympy/plotting/tests/test_plot.py
+++ b/sympy/plotting/tests/test_plot.py
@@ -91,7 +91,7 @@ def plot_and_save(name):
 
     p = plot(Piecewise((1, x > 0), (0, True)),(x,-1,1))
     p.save(tmp_file('%s_plot_piecewise' % name))
-    
+
     #parametric 2d plots.
     #Single plot with default range.
     plot_parametric(sin(x), cos(x)).save(tmp_file())

